docs/general/electronics.md

```diff
@@ -0,0 +1,123 @@
+
+| Wire & cable   | US/Canada                   | Europe & UK   | China & Russia | Oceania       | Japan         | India         |
+| -------------- | --------------------------- | ------------- | -------------- | ------------- | ------------- | ------------- |
+| Live / Line    | ![black wire] ![red wire]   | ![brown wire] | ![yellow wire] | ![red wire]   | ![black wire] | ![red wire]   |
+| Neutral        | ![white wire] ![grey wire]  | ![blue wire]  | ![black wire]  | ![black wire] | ![white wire] | ![black wire] |
+| Ground / Earth | ![green wire] ![earth wire] | ![earth wire] | ![earth wire]  | ![green wire] | ![green wire] | ![earth wire] |
+
+### DC wire color conventions
+
+There is no general standard for what colours are used for what voltage node, but there are some key standard colours we like to abide by in the 3D-printing community (derived from the ATX colour coding standard).
+
+| Potential | Symbol                     | Colour                        | Note                               |
+| --------- | -------------------------- | ----------------------------- | ---------------------------------- |
+| Ground    | `GND`                      | ![black wire]                 |                                    |
+| 5V        | `5V`                       | ![red wire]                   | For supply lines                   |
+| Signals   | `S`, `Signal`              | ![white wire]                 | For signals such as limit switches |
+| 12V       | `12V`                      | ![yellow wire]                |                                    |
+| 24V       | `24V`, `VCC`, `VMOT`, etc. | ![yellow wire]                |                                    |
+| 48V       | `48V`, `HT`, `VMOT`, etc.  | ![purple wire] ![orange wire] | Use a color different from 5V/24V  |
+
+## Wiring
+
+{: .danger }
+> Using smaller wires than recommended for their rated current may result in cables melting or catching fire. Always use a wire gauge that at least matches the current load!
+
+Using appropriately sized wires ensures the cables can withstand the current draw on the wire. Using a too small wire gauge will increase the wire resistance and cause the cables to (over) heat. 
+
+### Recommended wire gauges
+
+This table specifies recommended minimum gauges for nickle-free stranded copper wires; thicker wire can always be used. The rated ampacities are just a rule of thumb. At the rated limits the voltage drop (increases over distance), insulation temperature limit, thickness, thermal conductivity, and air convection and temperature should all be taken into account.
+
+| Application    | AWG     | Metric   | Max current rating |
+| -------------- | ------- | -------- | ------------------ |
+| Mains / AC     | 18 AWG  | 0.75 mm² | 10A @ 120/240VAC   |
+| Hotend heater  | 20 AWG  | 0.50 mm² | 6A @ 12-24VDC      |
+| Stepper motors | 22 AWG  | 0.32 mm² | 4A @ 12-48VDC      |
+| Signal wires   | 24 AWG  | 0.20 mm² | 2A @ 5-24VDC       |
```

Comment:
Should we add a line for other general DC power situations such as LED strips, lighting, fan wire extensions, etc?

Reply:
No